.github/workflows/profiling.yaml

```diff
@@ -0,0 +1,120 @@
+---
+name: Performance Profiling
+on:
+  # Trigger on Issue created; Run against master branch
+  issues:
+    types: [opened, edited]
+    branches: 
+      - 'master'
+  
+  pull_request:
```

Comment:
Does this mean it can be triggered on PRs as well?

Reply:
A PR and an Issue from GH point of view are essentially the same object type. So a comment in a PR and a comment in an Issue both cause this to trigger. And check our conditions